src/Components/Publishing/Display/track-once.ts

```diff
@@ -21,12 +21,12 @@ const trackOnce = (unitLayout, action) => {
   return (target, name, descriptor) => {
     const decoratedFn = descriptor.value
     // tslint:disable-next-line:only-arrow-functions
-    descriptor.value = function () {
+    descriptor.value = function() {
       const key = [
         this.props.campaign.name,
         unitLayout(this.props),
         action,
-        (this.props.article && this.props.article.id) || this._reactInternalInstance._debugID,
```

Comment:
@kanaabe - just curious how you ended up with `this._reactInternalInstance._debugID` to begin with?